alaoweb/2021.alaoweb.org
# Session links & times

* mostly addresses #171
* added Zoom & YouTube links to sessions.yml
* added start times & Q&A times
* to test on localhost: 
  * in config.yml set `showSessions: true` (line 278) 
  * `bundle exec jekyll build` & `bundle exec jekyll serve`
  * open a session on the schedule page and you should see the video as well as red "launch session" button
